File: 1013-FibonacciNumber/1013-FibonacciNumber.py
# Last updated: 8/13/2025, 8:18:05 PM
class Solution(object):
    def fib(self, n):
        if n==0 or n==1 :
            return n
        a = self.fib(n-1)
        b = self.fib(n-2)
        return a + b

File: 1013-FibonacciNumber/1013-FibonacciNumber.java
// Last updated: 8/13/2025, 8:18:13 PM
class Solution {
    public int fib(int n) {
        if(n==1 || n==0){
            return n;
        }
        int a = fib(n-1);
        int b = fib(n-2);
        return a+b;
    }
}